pkg/v1/cli/command/core/doc.go

```diff
@@ -39,17 +40,25 @@ var genAllDocsCmd = &cobra.Command{
 			return fmt.Errorf("error generate core tanzu cmd markdown %q", err)
 		}
 
-		plugins, err := cli.ListPlugins()
```

Comment:
We should continue to use this conditionally `config.IsFeatureActivated(config.FeatureContextAwareCLIForPlugins)` similar to what we are doing in other places until we completely remove deprecated code?

Reply:
changes are done to use conditional flag.

Reply:
@chandrareddyp what I mean here is to continue using `ListPlugins` when featureflag is false and not remove it completely to align the code with other conditional changes.

```
if !config.IsFeatureActivated(config.FeatureContextAwareCLIForPlugins){
    pluginDescriptions, err := cli.ListPlugins()
} else {
    pluginDescriptions, err = pluginmanager.InstalledPluginsDescriptors() 
}
```

Reply:
got it. will update the code. thanks.